tasks/plugins.yml

```diff
@@ -11,5 +11,5 @@
     ./bin/logstash-plugin install {{ item }}
     chdir=/opt/logstash
   with_items: "{{ logstash_install_plugins }}"
-  when: "item not in logstash_plugins_list.stdout"
+  when: "logstash_plugins_list is defined and item not in logstash_plugins_list.stdout"
```

Comment:
I think it would be better to add a `check_mode: no` to the task above this one, so it is always run. See the docs here: http://docs.ansible.com/ansible/latest/playbooks_checkmode.html

Reply:
If ./bin/logstash-plugin is not yet installed, check_mode: no will produce an error on the task above...